src/connectors/current-refinements/__tests__/connectCurrentRefinements-test.ts

```diff
@@ -547,5 +557,109 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/current-ref
         })
       );
     });
+
+    it('provides the items from multiple scoped results', () => {
+      const rendering = jest.fn();
+      const customCurrentRefinements = connectCurrentRefinements(rendering);
+      const widget = customCurrentRefinements({});
+
+      helper.addFacetRefinement('facet1', 'facetValue');
+
+      widget.init!(
+        createInitOptions({
+          helper,
+          state: helper.state,
+        })
+      );
+
+      helper
+        .addFacetRefinement('facet1', 'facetValue')
+        .addFacetRefinement('facet2', 'facetValue');
+
+      widget.render!(
+        createRenderOptions({
+          scopedResults: [
+            {
+              indexId: 'firstIndex',
+              helper,
+              results: new SearchResults(helper.state, [
+                createSingleSearchResponse({
+                  index: 'firstIndex',
+                }),
+              ]),
+            },
+            {
+              indexId: 'secondIndex',
+              helper,
+              results: new SearchResults(helper.state, [
+                createSingleSearchResponse({
+                  index: 'secondIndex',
+                }),
+              ]),
+            },
+          ],
+          state: helper.state,
+          helper,
+        })
+      );
+
+      const secondRenderingOptions = rendering.mock.calls[1][0];
+      const items: Item[] = secondRenderingOptions.items;
+      expect(items).toHaveLength(4);
+      expect(items).toEqual([
+        {
+          attribute: 'facet1',
+          label: 'facet1',
+          refinements: [
+            {
+              attribute: 'facet1',
+              label: 'facetValue',
+              type: 'facet',
+              value: 'facetValue',
+            },
+          ],
+          refine: expect.any(Function),
+        },
+        {
+          attribute: 'facet2',
+          label: 'facet2',
+          refinements: [
+            {
+              attribute: 'facet2',
+              label: 'facetValue',
+              type: 'facet',
+              value: 'facetValue',
+            },
+          ],
+          refine: expect.any(Function),
+        },
+        {
+          attribute: 'facet1',
+          label: 'facet1',
```

Comment:
aren't these duplicate? This will mean we can no longer use `attribute` or `label` as key

Reply:
Can you be more explicit by "duplicate"?

If you mean UI duplicate, this is what we already do in React InstantSearch and what you plan to do in the first iteration for InstantSearch.js too. We don't plan to prefix the label by the index name because it's not useful from the user POV.

Note that it is rather unlikely that you've got the same attribute in nested indices.

Reply:
I mean totally  identical. We can't avoid showing them twice in the UI, but we can avoid React errors if we have enough information to make them unique for the key

Reply:
Right, that makes sense! I'll add the `indexName` key to the refinement items.

Reply:
Fixed f994a628ca7d29896e6cd25c2219a1c6ec4eff5f.